scripts/helper_functions.sh

```diff
@@ -2,8 +2,12 @@
 
 # Common functions definitions
 
+function get_php_version {
+    local PHPVER=`/usr/bin/php -r "echo PHP_VERSION;" | /usr/bin/cut -c 1,2,3`
+    echo ${PHPVER}
+}
+
 function install_php_sql_driver 
-{
```

Comment:
Why is this line removed? Wouldn't this removal cause a syntax error at run time? I know it's not very easy to dev-test these scripts, but at least you can set up a full CI on your master and exercise any script changes on a full deployment? See @roperto 's recent work https://github.com/Azure/Moodle/pull/94.

I think this is a bug and must be fixed before merging.